contest/templates/contest/editContest.html

```diff
@@ -0,0 +1,63 @@
+<!--
```

Comment:
In edit contest page, contest owner should not appear as a candidate of coowner.